modules/aruco/test/test_aruco_tutorial.cpp

```diff
@@ -8,7 +8,7 @@
 namespace opencv_test { namespace {
 
 
-TEST(CV_ArucoTutorial, can_find_singlemarkersoriginal)
+TEST(DISABLED_CV_ArucoTutorial, can_find_singlemarkersoriginal)
 {
     string img_path = cvtest::findDataFile("aruco/singlemarkersoriginal.jpg", false);
```

Comment:
No need to disable, need to use testing API properly.

These files are part of opencv_extra. They can't be "optional" as specified by the second parameter. "optional" is used for files from downloaded shared drive (e.g. DL models).

`false` should be removed (removed not replaced).

Reply:
removed false